lib/utils/sendTx.js

```diff
@@ -12,9 +13,15 @@ export const sendTx = async (
   params = [],
   txDescription
 ) => {
-  setTx((tx) => ({
+  if (!walletMatchesNetwork) {
+    console.log(provider)
```

Comment:
nit: delete log

Reply:
We should make this `sendTx` fn be returned from `useSendTx` like `pool-app` and then just do the `useNetwork()` call in there, rather than funnelling down `walletMatchesNetwork` on every call of `sendTx`

Reply:
Agreed. I'll make a Linear story to do that soon.